doc/BUILD.rst

```diff
@@ -0,0 +1,500 @@
+Taisei Project - Build/Maintainer FAQ
+=====================================
+
+This is intended for anyone looking to build or package Taisei Project
+for any of its supported operating systems, including Linux, macOS, Windows,
+Emscripten, Switch, and others.
+
+.. contents::
+
+Dependencies
+------------
+
+Taisei's general dependencies are as follows:
+
+-  C (C11) compiler (``gcc``, ``clang``, etc)
+-  Python >= 3.5
+-  meson >= 0.53.0 (0.56.2 recommended)
+-  ninja >= 1.10.0
+-  docutils
+
+You can optionally install `other dependencies <#platform-specific-tips>`__,
+but the build system will pull in everything else you might need otherwise.
+
+Linux (Debian/Ubuntu)
+"""""""""""""""""""""
+
+On an apt-based system (Debian/Ubuntu), ensure you have build dependencies
+installed:
+
+.. code:: sh
+
+   apt-get install meson cmake build-essential
+   apt-get install libsdl2-dev libsdl2-mixer-dev libogg-dev libopusfile-dev libpng-dev libzip-dev libx11-dev
+
+If your distribution of Linux uses Wayland as its default window server, ensure
+that Wayland deps are installed:
+
+.. code:: sh
+
+   apt-get install libwayland-dev
+
+For packaging, your best bet is ``.zip``. Invoke ``ninja`` to package a
```

Comment:
I don’t understand. Does `ninja zip` put the whole game into an archive? Do I want to do this as a package maintainer? Why is it my best bet?

Reply:
Left over from when this was a development-focused doc, mostly as a "this is how you zip it up" example. I'm going to make it clearer in the `Packaging` section later.